added readme (#3)
EugeneSinica/cinema-app: README.md

# 🔴 Cinema App

> This project has a primitive functionality, but is written using all modern Java 

Reviewer: ```suggestion
This project uses the most popular modern Java frameworks an also supports REST principles.
```
don't be shy)

software development technologies.

The main goal of the app is to create a user, let the user make order (buy tickets)
to a certain movie session in a certain cinema hall.

By default, the app has two roles: USER and ADMIN.

> User can register, login and make orders. Admin can add movies, movie sessions, cinema halls.

Reviewer: let's describe functionality in more details

which endpoints are available to each role and what do they do

# 🔵 Technologies used:

> * Spring (Web MVC, Security, ORM)

Reviewer: ```suggestion
* Spring (Web MVC, Security, Core)
```

* Hibernate (Core, Validator)
* MySQL

> * Tomcat

Reviewer: ```suggestion
* Tomcat
* Maven
```

# 🚩 How to run the app

> First, clone the repo on GitHub. Second, open Intellij IDEA and open the project. Then configure Tomcat (fix war-exploded).

Reviewer: ```suggestion
First, clone the repo on GitHub. Second, open Intellij IDEA and open the project. Then configure Tomcat (choose war-exploded artifact).
```

> Also, you have to configure your database properties in main/resources/db.properties.

Reviewer: ```suggestion
Also, you have to configure your database properties in `main/resources/db.properties.`
```

# 🔴 Cinema App

> This project uses the most popular modern Java frameworks an also supports REST principles.

Reviewer: ```suggestion
This project uses the most popular modern Java frameworks and also supports REST principles.
```

# 🔴 Cinema App

This project uses the most popular modern Java frameworks an also supports REST principles.

The main goal of the app is to create a user, let the user make order (buy tickets)
to a certain movie session in a certain cinema hall.

By default, the app has two roles: USER and ADMIN.
User can register, login and make orders. Admin can add movies, movie sessions, cinema halls.

Available for **all** roles: /register;

**FOR USER:**

**Get:** /cinema-halls, /movies, /movie-sessions/* , /orders, /shopping-carts;

**Post:** /orders, /shopping-carts;

**FOR ADMIN:**

**Get:** /cinema-halls, /movies, /movie-sessions/**, /users/;

> **Post:** /cinema-halls, /movies, /movie-sessions/, /orders/, /shopping-carts;

Reviewer: are /orders and /shopping-carts available for admin?
